### PR TITLE
WIP fix: tests

### PR DESCRIPTION
**Pull request by ldicarlo**
Hello @hsyl20 , would these fixes be ok with you ? 
If yes I'll try do the other modules in another PR. 

(Following #46)

### CODE REVIEW

**Comment by ldicarlo** [comment]
That's all for `haskus-utils-variant`, I guess.

File: haskus-utils-variant/src/lib/Haskus/Utils/EGADT.hs [hunk 1]
```diff
@@ -48,24 +49,24 @@ import Haskus.Utils.Types
 -- >>>
 -- >>> :}
 --
--- >>> let y = VF @(AST Int) (VarF "a")
+-- >>> let y = Haskus.Utils.EGADT.VF @(AST Int) (VarF "a")
```
Comment (ldicarlo) [inline haskus-utils-variant/src/lib/Haskus/Utils/EGADT.hs:52]: This is not the right way to do that, I guess?

Comment (hsyl20) [reply haskus-utils-variant/src/lib/Haskus/Utils/EGADT.hs:52]: That looks verbose indeed. Why is that needed?

Comment (ldicarlo) [reply haskus-utils-variant/src/lib/Haskus/Utils/EGADT.hs:52]: The compiler does not which `VF` to use (EADT or EGADT I think)

Comment (hsyl20) [reply haskus-utils-variant/src/lib/Haskus/Utils/EGADT.hs:52]: Ah indeed. I don't remember why I've used the same name for both. My bad.

Let's use the long names then

Comment (ldicarlo) [reply haskus-utils-variant/src/lib/Haskus/Utils/EGADT.hs:52]: Can we merge this one as is? 
I will make another one with the rest during the week end

**Comment by ldicarlo** [comment]
Maybe using `doctests-parallel`is better. I am trying

**Pull request closed without merging**